### PR TITLE
[Triton/Gluon] [CI] Add fused KDA decode kernel (conv1d + recurrence + gated RMSNorm)

### PR DESCRIPTION
**Pull request by mengfei-jiang**
Single Triton kernel fusing three KDA decode operations:
- causal_conv1d_update (Q + K + V)
- delta-rule recurrence with KDA lower-bounded sigmoid gate
- gated RMSNorm output normalization

Key design: CHUNK_V = V (single chunk) so b_o stays in registers, avoiding global store→load between recurrence and RMSNorm phases.

Files:
- _triton_kernels/.../fused_conv_recurrent_norm.py: kernel
- gated_delta_net/fused_kda_decode.py: Python wrapper
- op_tests/triton_tests/test_fused_kda_decode.py: unit tests

## Submission Checklist

- [x] Look over the contributing guidelines at https://github.com/ROCm/ROCm/blob/develop/CONTRIBUTING.md#pull-requests.

### CODE REVIEW

**Review by zejunchen-zejun — DISMISSED** [review]

**Review by valarLip — APPROVED** [review]

File: aiter/ops/triton/_triton_kernels/gated_delta_rule/decode/fused_conv_recurrent_norm_unified.py [hunk 1]
```diff
@@ -0,0 +1,388 @@
...
+
+Functional-first — correctness over performance.
+"""
+
+import triton
+import triton.language as tl
+
+
+@triton.jit
+def _fused_kda_decode_unified_kernel(
```
[the remark this answers is not in the document]
Comment (Boss2002n) [reply aiter/ops/triton/_triton_kernels/gated_delta_rule/decode/fused_conv_recurrent_norm_unified.py:25]: @valarLip this is something we are trying to enforce on every new kernel - please don't merge kernels without this

File: aiter/ops/triton/_triton_kernels/gated_delta_rule/decode/fused_conv_recurrent_norm.py [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+
+Single Triton kernel. Grid: (batch, heads).
+"""
+
+import triton
+import triton.language as tl
+
+
+@triton.jit
+def fused_conv_recurrent_norm_kernel(
```
[the remark this answers is not in the document]
Comment (Boss2002n) [reply aiter/ops/triton/_triton_kernels/gated_delta_rule/decode/fused_conv_recurrent_norm.py:14]: @valarLip this is something we are trying to enforce on every new kernel - please don't merge kernels without this

File: op_tests/op_benchmarks/triton/bench_fused_kda_decode.py [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+import argparse
+
+import torch
+import triton
+import triton.language as tl
+from einops import rearrange
+
+from aiter.ops.triton._triton_kernels.gated_delta_rule.decode.fused_sigmoid_gating_recurrent import (
+    fused_sigmoid_gating_delta_rule_update,
+)
```
[the remark this answers is not in the document]
Comment (Boss2002n) [reply op_tests/op_benchmarks/triton/bench_fused_kda_decode.py:26]: @valarLip this is something we are trying to enforce on every new kernel - please don't merge kernels without this

File: aiter/ops/triton/gated_delta_net/fused_kda_decode.py [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+        stride_cw_group=stride_cw_group,
+        stride_cw_width=stride_cw_width,
+        stride_cw_ch=stride_cw_ch,
+        stride_cs_slot=conv_state.stride(0),
+        stride_cs_dim=conv_state.stride(1),
+        stride_cs_pos=conv_state.stride(2),
+        stride_beta_tok=stride_beta_tok,
+        stride_og_tok=stride_og_tok,
+        stride_ssm_slot=ssm_state.stride(0),
+        num_warps=2 if get_arch() == "gfx942" else 4,
```
[the remark this answers is not in the document]
Comment (Boss2002n) [reply aiter/ops/triton/gated_delta_net/fused_kda_decode.py:115]: @valarLip this is something we are trying to enforce on every new kernel - please don't merge kernels without this

File: aiter/ops/triton/gated_delta_net/fused_kda_decode_unified.py [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+        stride_bufk_slot=s_bk_slot,
+        stride_bufk_hv=s_bk_hv,
+        stride_bufk_pos=s_bk_pos,
+        stride_bufu_slot=s_bu_slot,
+        stride_bufu_hv=s_bu_hv,
+        stride_bufu_pos=s_bu_pos,
+        stride_bufg_slot=s_bg_slot,
+        stride_bufg_hv=s_bg_hv,
+        stride_bufg_pos=s_bg_pos,
+        num_warps=2 if get_arch() == "gfx942" else 4,
```
[the remark this answers is not in the document]
Comment (Boss2002n) [reply aiter/ops/triton/gated_delta_net/fused_kda_decode_unified.py:186]: @valarLip this is something we are trying to enforce on every new kernel - please don't merge kernels without this

File: op_tests/triton_tests/test_fused_kda_decode.py [hunk 6]
```diff
@@ -0,0 +1,264 @@
+# SPDX-License-Identifier: MIT
+# Copyright (C) 2024-2026, Advanced Micro Devices, Inc. All rights reserved.
+
+"""Tests for fused KDA decode kernel (conv1d + recurrence + gated RMSNorm)."""
```
[the remark this answers is not in the document]
Comment (Boss2002n) [reply op_tests/triton_tests/test_fused_kda_decode.py:4]: @valarLip this is also needed
